docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst

```diff
@@ -0,0 +1,121 @@
+Create a JDBC source connector for MySQL
+==============================================
+
+The JDBC source connector pushes data from a relational database, such as MySQL, to Apache Kafka where can be transformed and read by multiple consumers. 
+
+.. Tip::
+
+    Sourcing data from a database into Apache Kafka decouples the database from the set of consumers. Once the data is in Apache Kafka, multiple applications can access it without adding any additional query overhead to the source database.
+
+.. _connect_jdbc_mysql_source_prereq:
+
+Prerequisites
+-------------
+
+To setup a JDBC source connector pointing to MySQL, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to collect the following information about the source MySQL database upfront:
+
+* ``MYSQL_HOST``: The database hostname
+* ``MYSQL_PORT``: The database port
+* ``MYSQL_USER``: The database user to connect
+* ``MYSQL_PASSWORD``: The database password for the ``MYSQL_USER``
+* ``MYSQL_DATABASE_NAME``: the database name
```

Comment:
I think is good to be consistent. you either start all the sentences with Caps or not.

Reply:
Fixed, thanks!